script.js

```diff
@@ -155,12 +155,31 @@ for (let i = 0; i < toggleProjectWindow.length; i += 1) {
   toggleProjectWindow[i].addEventListener('click', togglePopupWindow);
 }
 
+// Object to store the user data
+const userData = {
+  name: 'name',
+  email: 'email',
+  message: 'message',
+};
+
 // Check if the email is in lower case, if it isn't then dont submit the form
 document.getElementById('contact-form').addEventListener('submit', (event) => {
+  const name = document.getElementById('contact-me-name').value;
   const email = document.getElementById('contact-me-email').value;
-
+  const message = document.getElementById('contact-me-message').value;
   if (email !== email.toLowerCase()) {
     event.preventDefault();
     document.getElementById('email-error').classList.add('active');
+  } else { // If everything is ok, then store it on localStore
+    userData.name = name;
+    userData.email = email;
+    userData.message = message;
+    localStorage.setItem('userData', JSON.stringify(userData));
```

Comment:
- [ ] Hi. you really did a great job but the requirements say that every time an input field is being updated, the content in the local storage should be updated too. In your code, you instead set to local storage when the user submits the form which is good but not required.
![image](https://user-images.githubusercontent.com/37901469/168183354-051117b2-b69e-4211-8db1-b815d3af686d.png)